AlbanStelzle/BagBrother
# Implementation of Empty Slot Visibility Control

## Summary
This pull request implements the functionality to control the visibility of empty slots in bags based on user settings through the updated `Frame:IsShowingItem(bag, slot)` function.

## Changes Implemented
- Implemented a new setting for controlling the visibility of empty slots.
- Updated the `Frame:IsShowingItem(bag, slot)` function to handle the display logic of empty slots based on the new setting.

## How to Test
1. Navigate to the BagBrother options.
2. Locate the new setting for 'Hidding empty slots'.
3. Toggle the setting to hide or show empty slots in bags, keyring or both (or none).
4. Verify that the empty slots are hidden or displayed based on the setting.

## Impact
This feature allows for a clean, minimized inventory

## Additional Information
- Related Issue: [#1436](https://github.com/Jaliborc/Bagnon/issues/1436)
